src/controllers.php

```diff
@@ -26,33 +26,22 @@
 
 // Specific event
 $app->get('/event/{id}', function ($id) use ($app) {
-    $event = $app['meetup']->getEvent($id);
 
     /** @var RandomService $randomService */
     $randomService = $app['random'];
 
-    $client = new Client();
-    $checkins = array_filter($client->lrange('checkin_'.$id, 0, 300));
-
-    $winners = (count($checkins) > 0)? $randomService->getRandomNumbers(count($checkins)) : array();
-    return $app['twig']->render(
-        'event.html.twig',
-        array('event' => $event, 'winners' => $winners, 'checkins' => $checkins)
-    );
-})->bind('event');
-
-// Check-in page for Event
-$app->get('/event/{id}/checkin', function ($id, Request $request) use ($app) {
-
     $event = $app['meetup']->getEvent($id);
     $client = new Client();
     $checkins = array_filter($client->lrange('checkin_'.$id, 0, 300));
 
+    $winners = (count($checkins) > 0) ? $randomService->getRandomNumbers(count($checkins)) : [];
```

Comment:
I see that you just copied that line from the other method but we could also void calling `count($checkins)` twice by introducing a variable or letting `RandomService` handle that empty validation.